src/components/DatePicker/DatePicker.tsx

```diff
@@ -22,7 +22,8 @@ import {monthName} from './utilities';
 import {Month} from './components';
 import styles from './DatePicker.scss';
 
-export {Range, Months, Year};
+export {Months};
```

Comment:
What's going on here? Is `Months` not a type even though it's imported from the same place as Year and Range?

Reply:
Bingo. Months is an enum, while Range and Year are both types. See https://github.com/Shopify/javascript-utilities/blob/master/src/dates.ts#L11


Follow up questions: is this annoying and would it be better if Months was a union type like `type Months = 0 |1|2|3|4|5|6|7|8|9|10|11`? Yeah probably, but thar be dragons in the javascript-utilities packages and I want to stop relying on eventually so just stick your fingers in your ears and ignore it for now :D